packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1438,9 +1469,16 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
     coordinator.updateCanDrag();
   }
 
-  void updateCanDrag(double totalExtent) {
-    context.setCanDrag(physics.allowUserScrolling &&
-        (totalExtent > (viewportDimension - maxScrollExtent) || minScrollExtent != maxScrollExtent));
+  void updateCanDrag(bool innerCanDrag,) {
```

Comment:
nit: that trailing comma feels a little lonly.

Reply:
Oh whoops. Thanks!